usaspending_api/conftest_helpers.py

```diff
@@ -78,6 +81,74 @@ def _generate_index_name(cls):
         )
 
 
+class TestElasticAwardSearchIndex:
```

Comment:
Either `TestElasticAwardSearchIndex` and `TestElasticSearchIndex` should be combined, or they should inherent a base class which contains the shared functions and attributes